src/pages/ReferralDetailsPage.js

```diff
@@ -64,15 +69,15 @@ function ReferralDetailsPage({route, account}) {
             headerContent={
                 <Icon
                     src={PaymentHands}
-                    width={178}
+                    width={isNative ? windowWidth : windowWidth * 2}
                     height={232}
                 />
             }
             headerContainerStyles={[styles.staticHeaderImage, styles.justifyContentEnd]}
             backgroundColor={theme.PAGE_THEMES[SCREENS.RIGHT_MODAL.REFERRAL].backgroundColor}
         >
-            <Text style={[styles.textHeadline, styles.mb3, styles.mt3, styles.ph4]}>{contentHeader}</Text>
-            <Text style={[styles.inlineSystemMessage, styles.ml0, styles.mb6, styles.ph4]}>{contentBody}</Text>
+            <Text style={[styles.textHeadline, styles.mb2, styles.ph5]}>{contentHeader}</Text>
+            <Text style={[styles.baseFontStyle, styles.ml0, styles.mb5, styles.ph5]}>{contentBody}</Text>
```

Comment:
This style is not accessible anymore in that way 

Working path - `styles.webViewStyles.baseFontStyle`